cmd/jag/commands/util.go

```diff
@@ -40,7 +49,10 @@ func GetSDK(ctx context.Context) (*SDK, error) {
 			return nil, err
 		}
 		if stat, err := os.Stat(sdkCachePath); err != nil || !stat.IsDir() {
-			return nil, fmt.Errorf("no SDK found in '%s', but Jaguar %s needs version %s.\nRun 'jag setup' to fix this.", sdkCachePath, info.Version, info.SDKVersion)
+			errorString := fmt.Errorf("no SDK found in '%s', but Jaguar %s needs version %s.", sdkCachePath, info.Version, info.SDKVersion)
+            fmt.Fprintf(os.Stderr, "%s.\n", errorString)
+            printFixLine()
+            return nil, errorString
```

Comment:
If you return the `errorString` error object, won't it be printed again as error message?